site/about page/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>about-page</title>
+  <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
+  <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
+  <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+
+<body>
+  <nav><a href="/">Home</a><a href="/about">About</a></nav>
+  <h1>Main</h1>
+  <div class="wrapper">
+    <div class="nav-1">
+      <div>
+        <h1>Language</h1>
+      </div>
+      <div>
+        <h2>Connect</h2>
+      </div>
+      <div>
+        <h2>Give</h2>
+      </div>
+      <div>
+        <h2>Apply</h2>
+      </div>
+    </div>
+    <div class="nav-2">
+      <div class="BA">
+        <h1>Boston Art</h1>
+        <h1>Academy</h1>
+      </div>
+      <div>
+        <h2>About</h2>
+      </div>
+      <div>
+        <h2>Academics</h2>
+      </div>
+      <div>
+        <h2>Admissions</h2>
+      </div>
+      <div>
+        <h2>Community</h2>
+      </div>
+      <div>
+        <h2>Events & Exhibitions</h2>
+      </div>
+      <div>
+        <h2>Founation</h2>
+      </div>
+    </div>
+    <div class="banner">
+      <h1>About</h1>
+    </div>
+    <div class="nav-3">
+      <div>
+        <h3>Mission</h3>
+      </div>
+      <div>
+        <h3>Our Story</h3>
+      </div>
+      <div>
+        <h3>New Campus</h3>
+      </div>
+      <div>
+        <h3>Our Graduates</h3>
+      </div>
+    </div>
+    <div class="bottom-page">
```

Comment:
Maybe try using a class like `our-mission`? Sometimes it can get confusing using classes like `bottom-page` when things start moving around.